### ISSUE TITLE
top_k_accuracy can't print true accuracy

### ISSUE THREAD

**Issue by RichardMrLu** [TropComplique/trained-ternary-quantization#3]
After I trained the ttq_densenet_small's model, I run the test_expolre file, but print top_k_accuracy as [0,0,0,0,0].
Then I use the model in the original file, 
print log_loss(val_true_targets, val_predictions) #2.70223670959
print accuracy_score(val_true_targets, val_predictions.argmax(1)) #0.3938
but print top_k_accuracy(val_true_targets, val_predictions, k=(2, 3, 4, 5, 10)) #[0,0,0,0,0]

How to solve this problem, Thanks for your answer.

**Comment by RichardMrLu** [comment]
and model.pytorch_state is 1.8M, model_ternary_quantization.pytorch_state is also 1.8M.. Maybe after quantizatied the model's memory should be smaller? I changed .ipynb file to .py file

**Comment by TropComplique** [comment]
Hi @RichardMrLu.
Please see this issue #2. 

I just tested `ttq_densenet_small/test_and_explore.ipynb` 
with pretrained model `ttq_densenet_small/model_ternary_quantization.pytorch_state`
using `pytorch 0.3`, it works. So you need to debug by yourself.

**Comment by RichardMrLu** [comment]
@TropComplique  I run it again on jupyter notebook, it work well. First time, I changed it to .py file, so caused some errors. Thanks for your help.
